libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -71,6 +72,14 @@ void UInt256Wrapper::Bind(Module &module)
       .EnableOperator(Operator::Equal)
       .EnableOperator(Operator::NotEqual)
       .EnableOperator(Operator::LessThan)
+      .EnableOperator(Operator::Add)
+      .EnableOperator(Operator::Subtract)
+      .EnableOperator(Operator::InplaceAdd)
+      .EnableOperator(Operator::InplaceSubtract)
+      .EnableOperator(Operator::Multiply)
+      .EnableOperator(Operator::Divide)
+      .EnableOperator(Operator::InplaceMultiply)
+      .EnableOperator(Operator::InplaceDivide)
       //        .EnableOperator(Operator::LessThanOrEqual)
```

Comment:
better remove those commented operators altogether or add a comment that they are tackled in another PR (and reference to that PR).

Reply:
done